### ISSUE TITLE
Concerning language preferences

### ISSUE THREAD

**Issue by JaroCamphuijsen** [NLeSC/guide#264]
https://github.com/NLeSC/guide/blob/a8adbd769e2d247e16ab9cd2b06aff3885be9354/best_practices/language_guides/languages_overview.md?plain=1#L24

While the Java section has been removed (at least temporarily), the language overview section still mentions a preference for "Java and Python over C++ and Perl". To me it looks strange to mention a preference for Java while not having the section in place to discuss it.

**Comment by Peter9192** [comment]
The list below that line is also outdated and doesn't correspond with the actual language chapters available currently.

**Comment by egpbos** [comment]
I agree that this statement is outdated. I would say that overwhelmingly, in projects where we start from scratch, we tend to prefer:

1. Python as a general purpose language that is easy to get started in, that can yield very good and efficient software and that is easily adoptable by many scientific communities.
2. JavaScript and TypeScript for web purposes.
3. C++ and CUDA for high performance code.

Other than that, we are flexible and able to adapt to the needs of the communities we want to reach in the projects we do.

**Comment by egpbos** [comment]
I'll add myself as page owner too.
